test/functional/test_framework/wallet.py

```diff
@@ -188,6 +188,35 @@ def send_to(self, *, from_node, scriptPubKey, amount, fee=1000):
         txid = self.sendrawtransaction(from_node=from_node, tx_hex=tx.serialize().hex())
         return txid, 1
 
+    def chain_tx(self, node, utxos, num_outputs=1, fee_per_output=1000):
```

Comment:
nit: (Some renames for symmetry)

```suggestion
    def send_self_transfer_multi(self, node_from, utxos, *, num_outputs=1, fee_per_output=1000):
```